### PR TITLE
update for semester WS16/17 (make jannikpeters commit run)

### PR DESCRIPTION
**Pull request by Miroka96**
Hopefully I clicked the right things
When will the official 180 will be up again? Who can I ask if you need help?

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -1262,11 +1315,24 @@ var data = {
...
+		kurz: "Softwaremodularität",
+		lehrform: ["Seminar"],
+		modul: ["Vertiefungsgebiete"],
+		semester: [ws16_17],
+		pflicht: false,
+		empfohlen: "",
+		vertiefung: [OSIS, SAMT],
+		cp: 3,
+		vorher: [],
+		dozent: ["Prof. Dr. Robert Hirschfeld ",],
```
Comment (knub) [inline js/data.js:1328]: Syntax error, trailing comma.

File: js/data.js [hunk 2]
```diff
@@ -1262,11 +1315,24 @@ var data = {
 		dozent: ["Prof. Dr. Christoph Meinel", "Ivonne Thomas"],
 		nameLV: "SOA- und Cloud-Standards, Praktische Anwendungen"
 	},
+	softwaremodlaritaet: {
```
Comment (knub) [inline js/data.js:1318]: Typo.

File: js/logic.js [hunk 3]
```diff
@@ -14,20 +14,20 @@ var studyRegulations = {
...
 	numberDisplayed: 6,
 	// current must be either lastSummerSemester or lastWinterSemester!
-	current: "SS16",
+	current: "WS16/17",
 	lastSummerSemester: "SS16",
-	lastWinterSemester: "WS15/16",
+	lastWinterSemester: "WS16/17",
 	/* the semester that is the first semester when you first start the application */
-	startswith: "WS15/16",
+	startswith: "SS17",
```
Comment (knub) [inline js/logic.js:30]: That's not possible. It cannot start with a semester that does not exist yet.
Also, it does not make sense to set this to a summer semester, as no HPI student starts his bachelor in the summer semester.

File: js/data.js [hunk 4]
```diff
@@ -135,7 +149,7 @@ var data = {
 		kurz: "BD Analytics",
 		lehrform: ["Vorlesung", "Übung"],
 		modul: ["Vertiefungsgebiete"],
-		semester: [ws15_16],
+		semester: [ws15_16, ws16_17],
```
Comment (knub) [inline js/data.js:152]: For all those cases, were you just added "ws16_17": Have you checked, that the Vertiefungsgebiete match the ones from the earlier semesters?

**Review by knub — CHANGES_REQUESTED** [review]

**Comment by knub** [comment]
> When will the official 180 will be up again? Who can I ask if you need help?

I do not know, I have no influence on that. You can try asking the FSR and/or the website club. Also, if you need it now, you can simply run it locally on your branch.

**Comment by Miroka96** [comment]
Maybe I have forked the wrong branch of jannikpeters... I will have a look at all changes and try some settings, also depending the new Studienordnung. Time to learn JavaScript.

**Comment by knub** [comment]
Closing as it's inactive.

**Pull request closed without merging**